src/Symfony/Component/Validator/Resources/translations/validators.sq.xlf

```diff
@@ -386,6 +386,50 @@
                 <source>This value is not a valid International Securities Identification Number (ISIN).</source>
                 <target>Kjo vlerë nuk është një numër i vlefshëm identifikues ndërkombëtar i sigurisë (ISIN).</target>
             </trans-unit>
+            <trans-unit id="100">
+                <source>This value should be a valid expression.</source>
+                <target>Kjo vlerë duhet të jetë një shprehje e vlefshme.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>This value is not a valid CSS color.</source>
+                <target>Kjo vlerë nuk është një ngjyrë e vlefshme CSS.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>This value is not a valid CIDR notation.</source>
+                <target>Kjo vlerë nuk është një shënim i vlefshëm CIDR.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
+                <target>Vlera e maskës së rrjetit duhet të jetë ndërmjet {{ min }} dhe {{ max }}.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Emri i skedarit është shumë i gjatë. Duhet të ketë maksimumi {{ filename_max_length }} karaktere ose më pak.|Emri i skedarit është shumë i gjatë. Duhet të ketë {{ filename_max_length }} karaktere ose më pak.</target>
```

Comment:
Can you double check this? Note the pattern
`singular | plural`. 

Does it say: "{{ filename_max_length }} character or less | {{ filename_max_length }} character**S** or less"?

Reply:
@Nyholm I probably have misapplied the translation from my friend (Albanian native speaker), I updated it again. Note that when you use translator, put the actual value `1` in singular variant, otherwise it gets translated as plural in both cases, most likely because of the placeholder `{{ filename_max_length }}`.